src/main/java/com/shopify/minishopify/controllers/CheckoutController.java

```diff
@@ -40,6 +42,8 @@ public ResponseEntity checkout(@RequestBody CheckoutList checkoutList) {
                 }
 
                 totalPrice += c.getQuantity() * p.getPrice();
+                p.setQuantity(p.getQuantity() - c.getQuantity());
+                productRepository.save(p);

```

Comment:
We should update the tests to check that this was persisted.

Reply:
Test updated